docs/_docs/modifiers/modifiers.md

```diff
@@ -270,42 +270,60 @@ interface HelloWorld {
     fun MainFrame(
         @Design(node = "#button") button: @Composable (ComponentReplacementContext) -> Unit,
     )
+    @DesignComponent(node = "#MyButton")
+    fun MyButton()
 }
 ```
 
-The example below passes a `Composable` `fun` that accepts
-`software.vehicle.design.ComponentReplacementContext`. This object provides
-methods to access the style of the replaced element and its descendants.
-Component Replacements are provided with information on the component they are
-replacing including the modifiers that would be applied for the layout style,
-visual style, and (if applicable) the text style.
-
-Doing so allows a replacement component to take on the designer specified
+The `software.vehicle.design.ComponentReplacementContext` object provides
```

Comment:
Is it still called `software.vehicle`? I thought it was `com.android.something` now...

Reply:
Whoops, good catch will fix.

Reply:
Done